cmd/odf/set/osd_settings.go

```diff
@@ -0,0 +1,46 @@
+package set
+
+import (
+	"fmt"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+var osdFullRatioCmd = &cobra.Command{
+	Use:                "set-full-ratio",
```

Comment:
How about a more general approach for the `set` commands? The command could be: 
```
odf set ceph <key> <value>
```

The key must be from a supported list, so we don't allow any ceph setting. This way it is much less code to add more keys that we want to allow setting.

Reply:
I made the changes but there is one more command that sets the recover profile as syntax
```
odf set recovery-profile <option>
```

may be we can move this command also under 'ceph' sub-command? as follow up pr?